contracts/cryptofighters/CryptoFightersV2.sol

```diff
@@ -130,7 +130,10 @@ contract CryptoFightersV2 is ERC721A, Ownable, ReentrancyGuard {
         Address.sendValue(payable(msg.sender), refundAmount);
     }
 
-    function refundGuaranteeActive() public view returns (bool) {
+    function getRefundGuaranteeEndBlock() public view returns (uint256) {
```

Comment:
`public refundEndTime` instead?

Reply:
More of a style choice. I feel get makes it clear it is a read from the caller POV.

Reply:
oh sorry I misunderstood. function has been renamed